src/composables/state.ts

```diff
@@ -0,0 +1,497 @@
+import VueCompositionAPI, { computed, ref } from '@vue/composition-api'
+import RoutingSlipService from '@/services/routingSlip.services'
+import { AccountInfo, AdjustRoutingSlipAmountPrams, AdjustRoutingSlipChequePrams, GetRoutingSlipRequestPayload, LinkedRoutingSlips, RoutingSlip, RoutingSlipDetails } from '@/models/RoutingSlip'
+import { ApiErrors, CreateRoutingSlipStatus, SlipStatus } from '@/util/constants'
+import CodesService from '@/services/codes.service'
+import CommonUtils from '@/util/common-util'
+import { BusinessInfo, GetFeeRequestParams, Payment, TransactionParams } from '@/models/Payment'
+import Vue from 'vue'
+import { Code } from '@/models/Code'
+
+Vue.use(VueCompositionAPI)
+
+// Routing Slips
+// Refs
+export const headerSearchTitle = ref([
```

Comment:
generally I think its better to import things like this into the store to keep it as small as possible

Reply:
Sounds good I'll make the change